docs/source/getting_started/writing_code.ipynb

```diff
@@ -7,10 +7,11 @@
    "source": [
```

Comment:
<p>"For security reasons, MAAP DPS jobs can only use code that is stored on the MAAP GitLab."</p><p>This is no longer true, any public repo can be used. Check with Sujen, George if this has changed to allow private repos in MAAP gitlab (possibly blocked by policy because all MAAP must be Open Source)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
fixed in b693af240d5db0c29671b718261adc9c84246ff6

Reply:
I think you should state: Any public git repository online can be used to host code.

Reply:
mentioned above in: 9ef281a4703bdb6bac778cc5e2e2f3700f53efde